worker/lease/dead_manager_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package lease_test
+
+import (
+	"reflect"
+
+	gc "gopkg.in/check.v1"
+
+	lease "github.com/juju/juju/worker/lease"
+)
+
+type DeadManagerSuite struct {
+}
+
+var _ = gc.Suite(&DeadManagerSuite{})
+
+type DeadManagerError struct{}
+
+const DeadManagerErrorMessage = "lease manager stopped"
```

Comment:
This error is the error message that is used to create the `DeadManager` using the `NewDeadManager(error) *Manager` method. However, not all methods that are called on a `DeadManger` return the error passed to `NewDeadManager`. With this PR applied some methods return the passed error and other return  and error with message "lease manager stopped". I made them the same for the purpose of submitting this PR with no failing tests. Ideally we the test should read "When DeadManager method is called, then the result will always be either the error passed to NewDeadManager or 'lease manager stopped'". That sentence, of course, contradicts the comment found above `NewDeadManager` which claims that all methods should return the passed in error. After fixing the panics the former behavior is what remains not the latter. Which behavior is correct/desired?